py_zipkin/thrift/__init__.py

```diff
@@ -146,6 +153,8 @@ def create_span(
         'binary_annotations': binary_annotations,
         'timestamp': int(timestamp_s * 1000000) if timestamp_s else None,
         'duration': int(duration_s * 1000000) if duration_s else None,
+        'trace_id_high': unsigned_hex_to_signed_int(trace_id_high)
+        if trace_id_high else None,
```

Comment:
i'm not sure if the the linter formatted it this way, but i would've indented this line more to make it clearer that it is part of the above line